views/default/css/elements/misc/spinner.css

```diff
@@ -0,0 +1,34 @@
+/* ***************************************
+	FIXED POSITION AJAX INDICATOR
+*************************************** */
+
+.elgg-spinner {
+    position: fixed;
+    top: 0;
+    left: 50%;
+    z-index: 10000;
+
+    box-sizing: border-box;
+    border: 1px solid #CCC;
+    border-top: 0;
+    border-radius: 0 0 10px 10px;
+
+    min-height: 0;
+    overflow: hidden;
+    width: 51px;
```

Comment:
I think it needs a bit more spacing. 55px or 61px for both width and height would make it more visible. White on white gets somewhat blended.